app/models/feature_flag.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class FeatureFlag
+  def self.yaml_to_hash(filename)
+    yaml = YAML.parse_file(Rails.root.join(filename))
+    hash = yaml.presence&.to_ruby || {} # Handle empty yaml file
+    hash.with_indifferent_access.freeze
+  end
+
+  DEFINITION = yaml_to_hash("app/config/feature_flags.yaml") # rubocop:disable Layout/ClassStructure
```

Comment:
I guess something like this would work, so you don't have to keep the method in the class object


```suggestion
  DEFINITION = begin
   yaml = YAML.parse_file(Rails.root.join("config/feature_flags.yaml"))
   hash = yaml.presence&.to_ruby || {}
   hash.with_indifferent_access
  end.freeze
```